library/core/class.pluginmanager.php

```diff
@@ -75,12 +80,13 @@ class Gdn_PluginManager extends Gdn_Pluggable implements ContainerInterface {
      * Initialize a new instance of the {@link Gdn_PluginManager} class.
      *
      * @param AddonManager $addonManager The addon manager that manages all of the addons.
+     * @param ContainerInterface $container The DI container.
      * @param EventManager $eventManager The event manager that handles all plugin events.
      */
-    public function __construct(AddonManager $addonManager = null, EventManager $eventManager = null) {
+    public function __construct(AddonManager $addonManager = null, ContainerInterface $container, EventManager $eventManager = null) {
```

Comment:
You've sandwiched a required parameter between two optional ones. I'd just use the container from the event manager.